src/cpdetect/compiler_detector/heuristics/pe_heuristics.cpp

```diff
@@ -327,6 +338,79 @@ void PeHeuristics::getVisualBasicHeuristics()
 	}
 }
 
+int32_t PeHeuristics::get_int32_unaligned(const uint8_t * codePtr)
+{
+	const int32_t * codePtrInt32 = reinterpret_cast<const int32_t *>(codePtr);
+
```

Comment:
Nitpick: Inconsistent use of tabs vs spaces (see the diff on GitHub, which makes it clear).